src/Mod/PartDesign/Gui/TaskPipeParameters.cpp

```diff
@@ -147,13 +149,17 @@ TaskPipeParameters::~TaskPipeParameters()
             PartDesign::Pipe* pipe = static_cast<PartDesign::Pipe*>(vp->getObject());
             Gui::Document* doc = vp->getDocument();
 
-            //make sure the user sees all important things: the
-            //spine/auxiliary spine he already selected
+            // hide/show the elements when they were hidden/show before the pipe was opened

```

Comment:
suggest:  
`// show/hide the elements when they were shown/hidden before the pipe was opened`